src/vsr/superblock_quorums.zig

```diff
@@ -97,7 +97,7 @@ pub fn QuorumsType(comptime options: Options) type {
 
             for (copies, 0..) |*copy, index| quorums.count_copy(copy, index, threshold);
 
-            std.sort.sort(Quorum, quorums.slice(), {}, sort_priority_descending);
```

Comment:
Unreleated, but I think `sort.insertion` actually would be good here, as the size is small, and we could save on codegen here. I wonder if should add `stdx.tiny_sorts` which asserts that the size of the slice is `<= 32`?